src/styles/blocks/header.scss

```diff
@@ -0,0 +1,82 @@
+@import '../utils/variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  padding: 0 50px;
+
+  &-logo {
+    height: 40px;
+    width: 40px;
+  }
+}
+
+// nav>list>list-item>list-item__link
+.nav {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+
+  &__list {
+    display: flex;
+    align-items: center;
+
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+
+    height: 60px;
+    margin: 0 10px;
+    padding: 0;
+
+    color: $dark-color;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    transition: color 300ms ease;
+
+    &:hover {
+      color: $blue-color;
+    }
+  }
+
+  &__item {
+    margin: 0;
+    padding: 0;
+
+    &:first-child .nav__link {
+      margin-left: 0;
+    }
+
+    &:last-child .nav__link {
+      margin-right: 0;
+    }
+  }
+}
+
+.is-active {
```

Comment:
The `.is-active` class should be modified to follow BEM conventions. If it's a modifier for the `.nav__link`, it should be named something like `.nav__link--active`.